torrents/management/commands/add_snatched_by_tracker.py

```diff
@@ -0,0 +1,180 @@
+                missing_data = True
+
+                #Make sure it isn't already seeding
+                if tracker_id in active_tracker_ids:
+                    continue
+                
+                torrent_info = fetch_torrent(realm, tracker, tracker_id, force_fetch=False)
+                torrent_file_bytes = bytes(torrent_info.torrent_file.torrent_file)
+                source_path = format_download_path_pattern(source_path_pattern, 
+                            torrent_file_bytes, torrent_info)
+                decoded = bdecode(torrent_file_bytes).get('info')
+                directory_name = decoded.get('name')
+                file_list = decoded.get('files')
+                file_paths = []
+                base_dir = os.path.join(source_path, directory_name)
+                #verify data exists at source_pattern
+                if file_list == None:
+                    # Single file case
+                    if os.path.exists(base_dir):
+                        missing_data = False
+                else:
+                    # Folder case
+                    for individual_file in file_list:
+                        file_path = ""
+                        
+                        # os.join all parts
+                        for each in individual_file['path']:
+                            # Bencode returns data in bytes if it is unicode
+                            # Decode it here
+                            if type(each) == bytes:
+                                each = each.decode('unicode_escape')
+                            file_path = os.path.join(file_path, each)
+
+                        
+                        
+                        individual_file_path = os.path.join(base_dir, file_path)
+                        if os.path.exists(individual_file_path):
+                            file_paths.append(file_path)
+                        
+                    if len(file_list) == len(file_paths):
+                        missing_data = False
+                if reject_missing and missing_data:
+                    continue
+
+                # Moves file to download_path, and adds torrent
+                itm = ImportedTorrentMover(source_path,file_paths, folder=directory_name, to_copy=to_copy)
+                added_torrent = add_torrent_from_tracker(
+                    tracker=tracker,
+                    tracker_id=tracker_id,
+                    download_path_pattern=download_path_pattern,
+                    force_fetch=True,
+                    store_files_hook=itm.move_files
+                )
+                
+            offset += limit
+            snatched = tracker.get_snatched(offset, limit)
+    
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--tracker_name',
+            help='Name of tracker, example: redacted',
+            type=str,
+            required=True
+        )        
+        parser.add_argument(
+            '--source_path_pattern',
```

Comment:
Please use --args-with-dashes-and-not-underscores. Python will still convert them to snake_case for you.